### PR TITLE
Point release for peppy

### PR DESCRIPTION
**Pull request by nsheff**
Due to the recent inadvertent caravel release we need to release peppy to support it.

Actually, probably good timing anyway.

### CODE REVIEW

**Comment by vreuter** [comment]
OK let me do `ubiquerg 0.0.3`

**Comment by nsheff** [comment]
caravel -> looper -> peppy -> ubiquerg.

funny

**Comment by vreuter** [comment]
OK `ubiquerg` has been released

File: docs/changelog.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Changelog
 
-## *v0.20* (2019-04-17):
+## [0.21.0] -- (2019-05-02)
+
+### Added
+- Support for Snakemake projects
+
```
Comment (nsheff) [inline docs/changelog.md]: Is there anything else we need to add to changelog @vreuter for this release?

File: peppy/sample.py [hunk 2]
```diff
@@ -15,19 +14,23 @@
...
-from .const import \
-    ALL_INPUTS_ATTR_NAME, DATA_SOURCE_COLNAME, DATA_SOURCES_SECTION, \
-    NAME_TABLE_ATTR, REQUIRED_INPUTS_ATTR_NAME, SAMPLE_EXECUTION_TOGGLE, \
-    SAMPLE_SUBANNOTATIONS_KEY, VALID_READ_TYPES
-from .utils import check_bam, check_fastq, copy, get_file_size, \
+from .const import *
+from .const import SNAKEMAKE_SAMPLE_COL
+from .utils import copy, get_file_size, get_logger, get_name_depr_msg, \
     grab_project_data, parse_ftype, sample_folder
+from ngstk import peek_read_lengths_and_paired_counts_from_bam
```
Comment (nsheff) [inline peppy/sample.py]: the new ngstk is showing up here already!

 that is quite a function name!

Comment (nsheff) [reply peppy/sample.py]: wow you've already got it on pypi. cool.

**Comment by nsheff** [comment]
@michalstolarczyk ready for this?

File: docs/changelog.md (hunk 1 above)
Comment (vreuter) [reply docs/changelog.md]: Hmm let me double-check

File: peppy/sample.py (hunk 2 above)
Comment (vreuter) [reply peppy/sample.py]: I couldn't come up with anything both meaningful and short...`check_bam` made a lot of sense when used more locally, but I wanted a more descriptive name for export from the seq. utils package

Comment (vreuter) [reply peppy/sample.py]: OK https://github.com/pepkit/peppy/pull/301/commits/1ee00259c94315f422da1ad1c5efe8e99a74d1c0 notes some improvements and changes

**Pull request merged**